Signals/Dto/MissingValuePolicy/SpecificValueMissingValuePolicy.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Dto.Conversions;
+
+namespace Dto.MissingValuePolicy
+{
+    public class SpecificValueMissingValuePolicy : MissingValuePolicy
+    {
+        public object Value { get; set; }
+
+        public Quality Quality { get; set; }
+
+        [MapFromGenericDataType]
```

Comment:
Perhaps name could be improved. "GenericDataType" seems to abstract - perhaps something like <AutoMapFromSignalDataType> or just <SignalDataType>. We're showing enough magic (unity, unity interception, wcf, unity interception combined with wcf, nhibernate, nhibernate automapper,  ... ) already, so ... :)